eng/pipelines/core-splitting.yml

```diff
@@ -0,0 +1,75 @@
+parameters:
+- name: OutputFolder
+  type: string
+  default: "projects"
+- name: numOfTestsPreJob
+  type: number
+  default: 20
+steps:
+  - pwsh: |
+      dotnet build /t:ProjectDependsOn ./eng/service.proj `
+        /p:ServiceDirectory=* /p:TestTargetDependency="Azure.Core" `
+        /p:IncludeSrc=false /p:IncludeStress=false `
+        /p:IncludePerf=false /p:RunApiCompat=false `
+        /p:InheritDocEnabled=false /p:BuildProjectReferences=false /p:BuildInParallel=true
+    displayName: Find all test dependencies
+
+  - pwsh: | 
+      $packages = Select-Xml -Path ./eng/projects.props -XPath '/Project/ItemGroup' | ForEach-Object { $_.Node.ProjectReference }
```

Comment:
If we are going to parse the list outside of msbuild we should just have msbuild output a very simple list instead of xml. That will make parsing easier here and you can then output the correct xml for msubuild in each of the output files.